LDAR_Sim/model_code/dev_satellite_company.py

```diff
@@ -0,0 +1,31 @@
+from methods.company import BaseCompany
+from methods.crew import BaseCrew
+
+
+class dev_satellite_company(BaseCompany):
```

Comment:
You can actually forgo using dev_satellite_company.py and just call dummy_company (P_dev_aircraft). In the future once we have all of the methods updated, we will remove dummy company.

Reply:
fixed